### PR TITLE
Translate to Korean #3563

### PR DESCRIPTION
**Pull request by nujabes403**
Added whole translation for Korean user. #3563

### CODE REVIEW

File: app/_locales/ko/messages.json [hunk 1]
```diff
@@ -1,10 +1,609 @@
...
+    "message": "잔액:"
+  },
+  "balanceIsInsufficientGas": {
+    "message": "가스가 충분하지 않습니다."
+  },
+  "beta": {
+    "message": "베타"
+  },
+  "betweenMinAndMax": {
+    "message": "1 달러 이상 2 달러 이하여야 합니다.",
```
Comment (kumavis) [inline app/_locales/ko/messages.json]: the 1 and 2 should be $1 and $2

**Review by kumavis — CHANGES_REQUESTED** [review]

File: app/_locales/ko/messages.json [hunk 2]
```diff
@@ -1,10 +1,609 @@
...
+    "message": "새 DEN 암호화"
+  },
+  "enterPassword": {
+    "message": "패스워드를 입력해주세요."
+  },
+  "etherscanView": {
+    "message": "이더스캔에서 계좌보기"
+  },
+  "exchangeRate": {
+    "message": "교환비율"
```
Comment (kumavis) [inline app/_locales/ko/messages.json]: jee.choi suggested `환율`

File: app/_locales/ko/messages.json [hunk 3]
```diff
@@ -1,10 +1,609 @@
...
+  },
+  "address": {
+    "message": "주소"
+  },
+  "addToken": {
+    "message": "토큰 추가"
+  },
+  "amount": {
+    "message": "양"
+  },
```
Comment (JeeChoi) [inline app/_locales/ko/messages.json]: The korean word does mean amount but is also the same word for sheep and I don't really see this word being used all the often. I would change it to 금액. same with amountPlusGas

File: app/_locales/ko/messages.json [hunk 4]
```diff
@@ -1,10 +1,609 @@
...
+  "appName": {
+    "message": "메타마스크",
+    "description": "어플리케이션 이름"
+  },
+  "attemptingConnect": {
+    "message": "블록체인에 접속 시도 중입니다."
+  },
+  "available": {
+    "message": "유효"
```
Comment (JeeChoi) [inline app/_locales/ko/messages.json]: I don't think this is the right word for available. It means 'valid' or 'of pertinence'

File: app/_locales/ko/messages.json [hunk 5]
```diff
@@ -1,10 +1,609 @@
...
+    "message": "비밀 키 (클릭하여 복사)"
+  },
+  "create": {
+    "message": "생성"
+  },
+  "createAccount": {
+    "message": "계좌 생성"
+  },
+  "createDen": {
+    "message": "생성"
```
Comment (JeeChoi) [inline app/_locales/ko/messages.json]: maybe should be 'DEN 생성.  Though 생성 is also an awkward word

File: app/_locales/ko/messages.json [hunk 6]
```diff
@@ -1,10 +1,609 @@
...
+  },
+  "crypto": {
+    "message": "암호화폐",
+    "description": "Exchange type (cryptocurrencies)"
+  },
+  "customGas": {
+    "message": "가스 설정"
```
Comment (JeeChoi) [inline app/_locales/ko/messages.json]: This says 'select Gas'

File: app/_locales/ko/messages.json (hunk 2 above)
Comment (JeeChoi) [inline app/_locales/ko/messages.json]: change to 환율 works better

File: app/_locales/ko/messages.json [hunk 7]
```diff
@@ -1,10 +1,609 @@
...
+    "message": "외부 비밀키"
+  },
+  "mainnet": {
+    "message": "이더리움 메인넷"
+  },
+  "message": {
+    "message": "메시지"
+  },
+  "min": {
+    "message": "최소"
```
Comment (JeeChoi) [inline app/_locales/ko/messages.json]: this says 'cancel' in korean

File: app/_locales/ko/messages.json [hunk 8]
```diff
@@ -1,10 +1,609 @@
...
+  "passwordMismatch": {
+    "message": "패스워드가 일치하지 않습니다.",
+    "description": "in password creation process, the two new password fields did not match"
+  },
+  "passwordShort": {
+    "message": "패스워드가 너무 짧습니다.",
+    "description": "in password creation process, the password is not long enough to be secure"
+  },
+  "pastePrivateKey": {
+    "message": "비밀키 스트링을 붙여넣어주세요.",
```
Comment (JeeChoi) [inline app/_locales/ko/messages.json]: I would change to "비밀키 입력해주세요" should roughly translates to 'please input private key'

File: app/_locales/ko/messages.json [hunk 9]
```diff
@@ -1,10 +1,609 @@
...
+    "message": "컨트랙트 승인"
+  },
+  "confirmPassword": {
+    "message": "패스워드 승인"
+  },
+  "confirmTransaction": {
+    "message": "트랜잭션 승인"
+  },
+  "continueToCoinbase": {
+    "message": "코인베이스로 계속하기"
```
Comment (JeeChoi) [inline app/_locales/ko/messages.json]: 코인베이스로 계속하기 should be 코인베이스로 계속가기

**Review by JeeChoi — COMMENTED** [review]
I don't know if it's important, but every where it says 'ether', it's been translated to 'ethereum' in Korean. For the most part it looks good. There were just a couple words that were awkward or aren't really used.

**Comment by nujabes403** [comment]
Thank you for review. I changed some word according to your comment.
@kumavis @JeeChoi

**Review by kumavis — APPROVED** [review]

**Comment by nujabes403** [comment]
@kumavis Thank you! It's really nice to be a member of Metamask team :)

**Pull request merged**